rust/crd/src/lib.rs

```diff
@@ -300,6 +309,20 @@ pub struct DruidClusterConfig {
     pub listener_class: CurrentlySupportedListenerClasses,
 }
 
+/// TODO: Better docs
```

Comment:
Could you please improve the docs a bit and link to https://github.com/kube-rs/kube/pull/1484? No need to write too much, will be removed in the future anyways

Reply:
You're right, I've updated the docs comment